tslint.json

```diff
@@ -0,0 +1,31 @@
+{
+  "rules": {
+    "class-name": true,
+    "comment-format": [true, "check-space"],
+    "indent": [true, "spaces"],
+    "no-duplicate-variable": true,
+    "no-eval": true,
+    "no-internal-module": true,
+    "no-trailing-whitespace": true,
+    "no-var-keyword": true,
+    "one-line": [true, "check-open-brace", "check-whitespace"],
```

Comment:
I don't know if all of us want to one line after 'open-brace'. What do you think? @hopkins-tk @elemementor @hotell